main.go

```diff
@@ -174,11 +185,21 @@ func runRecordings(conf *Config, camera *lepton3.Lepton3) error {
 
 		// If motion detected, allow minFrames more frames.
 		if motion.Detect(frame) {
-			if window.Active() {
+			writeLog := motionLogFrame <= totalFrames-(10*framesHz)
```

Comment:
instead of have multiple `motionLogFrame = totalFrames` below may do this here:
```go
if shouldLogMotion {
    motionLogFrame = totalFrames
}
```

That's means that future changes are less like to result in this being missed.